spec/supporting_spec.rb

```diff
@@ -1,9 +1,16 @@
 require 'spec_helper'
 
 describe "QboApi::Supporting" do
+  let(:api) do
+    if oauth2_creds[:access_token]
```

Comment:
Good idea on the `let(:api)`. Can you put the condition in a method within `spec/support/credentials.rb` and then have the `let` be assigned to this new method e.g. `let(:api) { get_creds }`